api/users.go

```diff
@@ -124,3 +127,15 @@ func (c *Client) LinkShadowUser(accessToken string, reqRealmName string, userID
 	_, err := c.post(accessToken, nil, url.Path(shadowUser), url.Param("realm", reqRealmName), url.Param("id", userID), url.Param("provider", provider), body.JSON(fedIDKC))
 	return err
 }
+
+// SendCPREmail sends an email to a user in the format definden in the CPR project
+func (c *Client) SendCPREmail(accessToken string, reqRealmName string, realmName string, cprEmailRep keycloak.CPREmailRepresentation) error {
```

Comment:
There is nothing specific to CPR. So we can have a generic comment and method name